topics/ui/index.md

```diff
@@ -0,0 +1,10 @@
+---
+aliases: user-interface
+created_by: Glenda Schroeder
+display_name: User interface
+released: '1985'
```

Comment:
In 1985 the `G`raphical `U`ser `I`nterface was introduced, but I think that `UI` was probably introduced long before. I think this is another instance of this topic being too broad to include a specific year like this, thoughts on removing it?
```suggestion
```